socobo-inventory.html

```diff
@@ -78,7 +78,7 @@
         @method _handleLogin
         */
         _handleLogin: function(login) {
-          return login.firebaseUrl + "inventory/" + login.userId + "/";
+          return login ? login.firebaseUrl + "inventory/" + login.userId + "/" : this.location;
```

Comment:
But the login object is not null?

Sebastian Kloppe <notifications@github.com> schrieb am Fr., 8. Jan. 2016,
15:55:

> In socobo-inventory.html
> <https://github.com/lyio/socobo-inventory/pull/26#discussion_r49195269>:
>
> > @@ -78,7 +78,7 @@
> >          @method _handleLogin
> >          */
> >          _handleLogin: function(login) {
> > -          return login.firebaseUrl + "inventory/" + login.userId + "/";
> > +          return login ? login.firebaseUrl + "inventory/" + login.userId + "/" : this.location;
>
> Looks good, but what if userId is null? After loggout The userId is null.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lyio/socobo-inventory/pull/26/files#r49195269>.
>